certified-connectors/EXPOCAD/apiDefinition.swagger.json

```diff
@@ -0,0 +1,4316 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "v3",
+    "title": "EXPOCAD",
+    "description": "This service interfaces with live EXPOCAD\u00ae event data.  Please use PUT, POST, and DELETE operations with discretion. Keep authorization codes private.  If your code has been compromised, contact tech@expocad.com immediately.",
```

Comment:
@mikefactorial Unicodes are not allowed, Can you please replace unicodes with respective English characters

Reply:
Hi @sriyen-msft this has been resolved in the latest commit